homeassistant/components/azuredns/__init__.py

```diff
@@ -0,0 +1,176 @@
+"""Support for Azure DNS services."""
+import logging
+
+from datetime import datetime
+from datetime import timedelta
+
+import requests
+import aiohttp
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_HOST, CONF_DOMAIN
+from homeassistant.helpers.event import async_track_time_interval
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+REQUIREMENTS = ['adal==1.2.1']
```

Comment:
We no longer define `REQUIREMENTS` inside the Python file. It's in `manifest.json`